### PR TITLE
[TabView] Update high contrast colors | MicaAlt colors - update high contrast colors

### PR DESCRIPTION
**Pull request by tashatitova**
Did a high contrast pass and fixed some bugs on tabs, selected tab, outlines, tab close button.
Added a border for the close button in high contrast as usually required by accessibility.

Updated MicaAlt colors to proper background in high contrast as these are usually backgrounds and we want to avoid flooding large areas with highlight color.

Before:
before

Updated, multiple hc themes:

https://user-images.githubusercontent.com/29714167/156718233-a3054d93-cbf5-44bf-89a2-cb73667f0104.mp4

### CODE REVIEW

**Review by marksfoster — APPROVED** [review]

**Comment by beervoley** [comment]
:shipit:

**Review by beervoley — APPROVED** [review]

**Comment by karenbtlai** [comment]
>             <StaticResource x:Key="TabViewBackground"                                       ResourceKey="SubtleFillColorTransparentBrush" />

This should be `SubtleFillColorTransparentBrush` as part of the standard to avoid using System* brushes for light and dark themes.

---
Refers to: dev/TabView/TabView_themeresources.xaml:9 in fbfd705. [](commit_id = fbfd7059192369216adce37cefe1cc11546c7f6b, deletion_comment = False)

File: dev/TabView/TabView_themeresources.xaml [hunk 1]
```diff
@@ -138,60 +150,66 @@
...
+            <StaticResource x:Key="TabViewItemHeaderCloseButtonBorderBrushDisabled"         ResourceKey="SystemControlTransparentBrush" />
 
             <!-- Note: These theme resources below are no longer used and might be removed in a future WinUI update. -->
             <StaticResource x:Key="TabViewButtonBackgroundActiveTab"                        ResourceKey="SystemControlTransparentBrush" />
             <StaticResource x:Key="TabViewButtonForegroundActiveTab"                        ResourceKey="SystemControlBackgroundBaseMediumBrush" />
 
-            <StaticResource x:Key="TabViewBorderBrush"                                      ResourceKey="CardStrokeColorDefault" />
+            <SolidColorBrush x:Key="TabViewBorderBrush"                                     Color="{ThemeResource SystemColorHighlightColor}" />
             <StaticResource x:Key="TabViewItemBorderBrush"                                  ResourceKey="SubtleFillColorTransparentBrush" />
+            <Thickness x:Key="TabViewItemHeaderCloseButtonBorderThickness">1</Thickness>
```
Comment (karenbtlai) [inline dev/TabView/TabView_themeresources.xaml:212]: >Thickness [](http://example.com/codeflow?start=78&length=9)

This should be moved to outside the resource dictionaries to where the rest of the thickness resources are.

**Review by karenbtlai — CHANGES_REQUESTED** [review]
Move Thickness Resources + update transparent brush in light/dark theme.

File: dev/TabView/TabView_themeresources.xaml (hunk 1 above)
Comment (tashatitova) [reply dev/TabView/TabView_themeresources.xaml:212]: @karenbtlai it's a theming resource.

**Review by karenbtlai — APPROVED** [review]

**Pull request merged**